### PR TITLE
Adsk Contrib - LUT tests

### PR DESCRIPTION
**Pull request by BernardLefebvre**
Port further LUT tests using CLF/CTF parsing.

### CODE REVIEW

File: src/OpenColorIO/ops/Lut1D/Lut1DOpGPU.cpp [hunk 1]
```diff
@@ -59,41 +60,41 @@ void PadLutChannels(unsigned long width,
         const unsigned long step = width - 1;
         for (unsigned long i = 0; i < (currWidth - step); i += step)
         {
-            chn.insert(chn.end(), &channel[3 * i], &channel[3 * (i + step)]);
+            std::transform(&channel[3 * i], &channel[3 * (i + step)], std::back_inserter(chn), [](float val) {return SanitizeFloat(val); });
```
[the remark this answers is not in the document]
Comment (BernardLefebvre) [reply src/OpenColorIO/ops/Lut1D/Lut1DOpGPU.cpp]: Next commit is splitting it.

File: src/OpenColorIO/ops/Lut1D/Lut1DOpGPU.cpp [hunk 2]
```diff
@@ -59,41 +60,41 @@ void PadLutChannels(unsigned long width,
...
             leftover -= step;
         }
 
         // If there are still texels to fill, add them to the texture data.
         if (leftover > 0)
         {
-            chn.insert(chn.end(),
-                       &channel[3 * (currWidth - leftover)],
-                       &channel[3 * (currWidth - 1)]);
+            std::transform(&channel[3 * (currWidth - leftover)], &channel[3 * (currWidth - 1)], std::back_inserter(chn), [](float val) {return SanitizeFloat(val); });
```
[the remark this answers is not in the document]
Comment (BernardLefebvre) [reply src/OpenColorIO/ops/Lut1D/Lut1DOpGPU.cpp]: Next commit is splitting it.

**Comment by hodoulp** [comment]
Following the two-weeks rule, here is a friendly reminder that the branch could be merged next week unless there are some objections.

**Comment by hodoulp** [comment]
24-hour reminder of the coming merge. Please have a look if you haven't yet.

**Pull request merged**